packages/sdk-react/src/ui/table/TablePagination.tsx

```diff
@@ -148,39 +141,41 @@ export const TablePagination = <T,>({
         >
           <ArrowRight fontSize="small" aria-label={t(i18n)`Next page`} />
         </IconButton>
-      </Grid>
-      {hasPageSizeSelect && (
-        <Grid
-          item
-          xs={2}
-          md={2}
-          lg={1}
-          display="flex"
-          justifyContent="flex-end"
-        >
-          <StyledSelect
-            {...slotProps?.pageSizeSelect}
-            aria-label={t(i18n)`Rows per page`}
-            MenuProps={{
-              ...slotProps?.pageSizeSelect?.MenuProps,
-              container: root,
+        {hasPageSizeSelect && (
+          <Box
+            sx={{
+              position: 'absolute',
```

Comment:
`position: absolute` could work, but `<StyledSelect />` could be configured using the theme. Therefore it is important not to break the layout.

Here is the overrided `<StyledSelect />` example:
<img width="1480" alt="image" src="https://github.com/team-monite/monite-sdk/assets/725645/07e91148-e35f-4510-a346-7b47bbc02a1b">

I would suggest only adding an extra blank cell to be used for centering.
Probably, usage of `<FormControl size="small"/>` also not needed

Reply:
fixed